.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Settings common to each job
+var_job_defaults: &job_defaults
+  working_directory: ~/verdaccio-google-cloud-storage
+  docker:
+    - image: cimg/node:lts-browsers
+
+orbs:
+  node: circleci/node@5.1.0
+  build-tools: circleci/build-tools@3.0.0
+
+commands:
+  # Command for checking out the source code from GitHub. This also ensures that the source code
+  # can be merged to the master branch without conflicts.
+  checkout_and_rebase:
+    description: Checkout and verify clean merge with master
+    steps:
+      - checkout
+      - build-tools/merge-with-parent:
+          parent: master
+
+  setup:
+    description: 'Setup Executor'
+    steps:
+      - attach_workspace:
+          at: ~/
+
+# Job definitions
+# See: https://circleci.com/docs/2.0/configuration-reference/#jobs
+jobs:
+  initialize:
+    <<: *job_defaults
+    steps:
+      - checkout_and_rebase
+      - node/install-packages
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - verdaccio-google-cloud-storage
+
+  lint:
+    <<: *job_defaults
+    steps:
+      - setup
+      - run:
+          name: Linting
+          command: npm run lint
+
+  type-check:
+    <<: *job_defaults
+    steps:
+      - setup
+      - run:
+          name: Type Checking
+          command: npm run type-check
+
+  unit-test:
+    <<: *job_defaults
+    steps:
+      - setup
+      - run:
+          name: Unit Testing
+          command: npm run test:unit
+
+
+# Invoke jobs via workflows
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
+  version: 2
```

Comment:
I believe this is deprecated in v2.1.

Reply:
You're right. The workflows version key is not required for v2.1 configuration.

Removed